src/Microsoft.AspNetCore.Server.Kestrel/Systemd/KesterlServerOptionsSystemdExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using Microsoft.AspNetCore.Server.Kestrel;
+
+namespace Microsoft.AspNetCore.Hosting
+{
+    public static class KesterlServerOptionsSystemdExtensions
```

Comment:
Technically the "System.Diagnostics.Process" dependency is added for netstandard, but that's in the shared runtime. Otherwise it's just one small static class (KesterlServerOptionsSystemdExtensions.cs) with two methods.